### DISCUSSION TITLE
set domain and configure lets encrypt SSL

### DISCUSSION THREAD

**Discussion by alez404** [telekom-security/tpotce#925] [Q&A]
hi\
how can i set a valid domain to show tpot on address insttead of ip:64297 address \
i wana set lets encrypt ssl or cloudflare ssl but first i should set a domain name and change the port of runnig t-pot from 64297 to something like 2807

**Comment by t3chn0m4g3** [comment]
1. Stop T-Pot (`systemctl stop tpot`)
2. Adjust `/opt/tpot/etc/tpot.yml` to your needs.
3. Since you are changing a host related port you need to change `/opt/tpot/bin/rules.sh` to reflect this as well.
4. `reboot`
